contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/file_excluder.py

```diff
@@ -24,13 +23,6 @@ def __init__(self, excludes_path, log):
             pattern, plugins = line.strip().split('::', 2)
             plugins = plugins.split()
 
-            deprecated_conditional(
-              lambda: 'pep8' in plugins,
```

Comment:
There's a conditional just below (with disturbing indent irregularity!) that can be removed now. I think in fact the whole style_plugins collection loop can be killed and `style_plugins = plugins.split()` above.